css/excercise_one/excercise_one.css

```diff
@@ -0,0 +1,65 @@
+* {
+  font-family: 'Poppins';
+  color: black;
+  background-color: white;
+  text-decoration: none;
+}
+
+nav{
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 25px;
+}
+
+.nav-link{
+  width: 294px;
+  height: 21px;
+  gap: 37px;
+  display: flex;
+  justify-content: space-between;
+  padding: 10px 30px;
+  align-items: center;
+}
+
+.nav-register{
+  width: 298px;
+  height: 33px;
+  gap: 34px;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+.logo{
+  width: 130.69px;
+  height: 43.31px;
+}
+
+.sign-up{
+  font-family: 'Poppins', sans-serif;
```

Comment:
don't think this is needed as you already have it above. same on the other classes that have this.

Reply:
I tried to remove it and I had problems, but didn't understand why as I used the *

Reply:
What is the problem you have? Check also that here you have the `sans-serif` while in the * you don't have it

Reply:
i believe you will need to import this on the HTML
`<link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">`